src/main/python/ttconv/tt.py

```diff
@@ -37,36 +37,81 @@
 import ttconv.srt.writer as srt_writer
 import ttconv.scc.reader as scc_reader
 
-LOGGER = logging.getLogger(__name__)
+LOGGER = logging.getLogger("ttconv")
 
 READING = True
 
-# Print iterations progress
-def print_progress_bar (iteration, total, prefix = '', suffix = '', decimals = 1, length = 100, fill = '█', print_end = "\r"):
+class ProgressConsoleHandler(logging.StreamHandler):
   """
-  Call in a loop to create terminal progress bar
-  @params:
-      iteration   - Required  : current iteration (Int)
-      total       - Required  : total iterations (Int)
-      prefix      - Optional  : prefix string (Str)
-      suffix      - Optional  : suffix string (Str)
-      decimals    - Optional  : positive number of decimals in percent complete (Int)
-      length      - Optional  : character length of bar (Int)
-      fill        - Optional  : bar fill character (Str)
-      print_end    - Optional  : end character (e.g. "\r", "\r\n") (Str)
+  A handler class which allows the cursor to stay on
+  one line for selected messages
   """
-  percent = ("{0:." + str(decimals) + "f}").format(100 * (iteration / float(total)))
-  filled_length = int(length * iteration // total)
-  bar_val = fill * filled_length + '-' * (length - filled_length)
-  print(f'\r{prefix} |{bar_val}| {percent}% {suffix}', end = print_end)
-  # Print New Line on Complete
-  if iteration == total: 
-    print()
+  is_writing_progress_bar = False
+  last_progress_msg = ""
```

Comment:
This should be set in the `__init__` method otherwise it is shared by all instances of the class.